contracts/mocks/MockSafe1271Caller.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import { IModule } from "contracts/interfaces/modules/IModule.sol";
+import { EncodedModuleTypes } from "contracts/lib/ModuleTypeLib.sol";
+import { PackedUserOperation } from "account-abstraction/contracts/interfaces/PackedUserOperation.sol";
+import { ECDSA } from "solady/src/utils/ECDSA.sol";
+import { SignatureCheckerLib } from "solady/src/utils/SignatureCheckerLib.sol";
+import { MessageHashUtils } from "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
+import "contracts/types/Constants.sol";
+import { IERC7579Account } from "contracts/interfaces/IERC7579Account.sol";
+
+contract MockSafe1271Caller is IModule {
```

Comment:
would this caller have it's own typehash?  = Dom Sep B ?
like we have some tests using TokenWithPermit.sol

Reply:
Yes, it will have it's own typehash in the wild 
Just not implemented here for sake of speed